matrix/plugins/actions.py

```diff
@@ -0,0 +1,45 @@
+from datetime import timedelta, datetime
+
+from .glitch import glitch  # noqa
```

Comment:
Do we need this import? I don't see glitch referenced in the Python code below ...